### DISCUSSION TITLE
Removal/deprecation of Latex documentation

### DISCUSSION THREAD

**Discussion by jpfeuffer** [OpenMS/OpenMS#6381] [Polls]
@OpenMS/core What do you think about removing the Latex documentation?
Latex is a HUGE dependency for all the build jobs and at every update, there are at least 2 package incompatibilities.

What advantages does it have, especially now that we translated the Tutorial to Markdown.

**Comment by poshul** [comment]
Does the Latex documentation have any content that we are missing elsewhere? Otherwise I'm amenable to ditching it.

**Reply by jpfeuffer** [reply]
Now that the User tutorial is gone, I think it is basically a secondary output format from Doxygen that we can output.
We might need to switch to MathJax for formulas inside class documentation (if there are any).

**Reply by jpfeuffer** [reply]
Installing latex with only the necessary packages from texlive already takes 20+ minutes on an empty docker image.

**Reply by jpfeuffer** [reply]
In theory, it might be much quicker now, since the needed packages need a significant update after the user tutorial was ported.

**Reply by jpfeuffer** [reply]
Note that MathJax will require JS on the user-side. There are still people refusing to use it.

**Comment by tjeerdijk** [comment]
I would also be in favor of ditching LaTeX. LaTeX predates a lot of modern software engineering and the TexLive “solution” of simply including every package under the sun leads to bloated software.

Also, we still have the technical writers on the payroll and can ask them to look into any holes left when we ditch LaTeX

On 12. Oct 2022, at 17:31, Julianus Pfeuffer ***@***.******@***.***>> wrote:



Note that MathJax will require JS on the user-side. There are still people refusing to use it.
